SECURITY.md

```diff
@@ -20,9 +20,14 @@ To ensure the timely response to your report, please ensure that the entirety
 of the report is contained within the email body and not solely behind a web
 link or an attachment.
 
-The lead maintainer will acknowledge your email within 48 hours, and will send a
-more detailed response within 48 hours indicating the next steps in handling
-your report. After the initial reply to your report, the security team will
+A [Security triage team member](https://github.com/expressjs/security-wg#security-triage-team) 
+or [the repo captain](https://github.com/expressjs/express/blob/master/Contributing.md#active-projects-and-captains) 
+will acknowledge your email within 5 working days, and will send a more detailed 
+response within 7 working days indicating the next steps in handling your report. 
```

Comment:
I don't think we should be making promises and/or setting timelines like this.    just say we will acknowledge them as soon as possible.  I also wouldn't promise to send them a details report, etc.

Reply:
we can/should have our own internal guidelines on vulnerability management which can include target timelines, but there's little/no benefit to codifying it in the security policy itself

Reply:
> we can/should have our own internal guidelines on vulnerability management which can include target timelines

In the oven https://github.com/expressjs/security-wg/pull/56

> but there's little/no benefit to codifying it in the security policy itself

Agree, I will remove it from here and include them only in the other guide/runbook

Reply:
Updated in 96bb4f6.